templates/jitsi_meet_config.js.j2

```diff
@@ -250,7 +250,7 @@ var config = {
     // minParticipants: 2,
 
     // Use XEP-0215 to fetch STUN and TURN servers.
-    useStunTurn: {{ jitsi_meet_use_stun_turn }},
+    useStunTurn: {{ jitsi_meet_use_stun_turn | to_json}},
```

Comment:
Here it would be useful add a space before closing the mustaches to make the Ansible lint happy

Reply:
I would suggest `| bool | lower ` instead of `| to_json `